src/Avalonia.X11/X11Window.cs

```diff
@@ -722,6 +723,10 @@ private void DispatchInput(RawInputEventArgs args)
         {
             if (_inputRoot is null)
                 return;
+
+            if (_disabled && args is RawPointerEventArgs pargs && pargs.Type == RawPointerEventType.Move)
```

Comment:
Are we filtering press/release elsewhere?

Reply:
The window doesn't gain input focus when disabled, so button press/receive events aren't passed to the window, but pointer move events are.

Reply:
what about PointerPressed/PointerReleased?

Reply:
They won't be sent to the window by the window manager when input is disabled.